Source/cielim/Private/RenderingFunctionsLibrary.cpp

```diff
@@ -25,4 +25,4 @@ void URenderingFunctionsLibrary::ApplyCosmicRays()
 void URenderingFunctionsLibrary::ApplyReadNoise()
 {
 	
-}
+}
```

Comment:
We want a single new line at the end of each file. This little red circle with a line through it indicates that the line is absent. 

For reference: https://gist.github.com/OleksiyRudenko/d51388345ea55767b7672307fe35adf3